src/main/java/com/ldtteam/structurize/client/fakelevel/BlueprintBlockAccess.java

```diff
@@ -47,7 +48,12 @@ public BlockState getBlockState(final BlockPos pos)
 
     public BlockState prepareBlockStateForRendering(final BlockState state, final BlockPos pos)
     {
-        if (Structurize.getConfig().getClient().renderPlaceholdersNice.get())
+        return prepareBlockStateForRendering(state, pos, null);
+    }
+
+    public BlockState prepareBlockStateForRendering(final BlockState state, final BlockPos pos, final BlueprintPreviewData previewData)
```

Comment:
Huh? Did i actually approved that in prev PR? the getBlockState(pos) must return correct blockState in as most as possible contextes - this should be done in similar way as `solidSubstitutionOverride`

Reply:
ye saw that too, I'll make it in an extra PR so I can merge that one to both versions